src/sst/core/sparseVectorMap.h

```diff
@@ -83,50 +117,131 @@ class SparseVectorMap {
     friend class ConfigGraph;
 
 public:
+
+    /**
+       Default constructor for SparseVectorMap
+     */
+    SparseVectorMap() {}
+
+    /**
+       Constructor that allows you to pass an already filled in array
+       with data.  The data in the passed in vector will be swapped
+       into the data vector of the sparsevectormap and the passed in
+       vector will be empty.
+
+       @param new_data Vector of data to swap into the sparsevectormap
+       data
+
+       @param sorted Specifies whether the vector is already sorted in
+       ascending order. if not, it will be sorted after swapping the
+       data in.
+    */
+    SparseVectorMap(std::vector<classT>& new_data, bool sorted = false)
+    {
+        data.swap(new_data);
+        if (!sorted) {
+            std::sort(data.begin(),data.end,
+                [] (const classT& lhs, const classT& rhs) -> bool
+                    {
+                        return lhs.key() < rhs.key();
+                    });
+        }
+    }
+
     typedef typename std::vector<classT>::iterator iterator;
     typedef typename std::vector<classT>::const_iterator const_iterator;
 
-    // Essentially insert with a hint to look at end first.  This is
+    // Essentially insert with a hint to look at end first.  this is
     // just here for backward compatibility for now.  Will be replaced
     // with insert() once things stabilize.
-    void push_back(const classT& val)
+    classT& push_back(const classT& val) __attribute__ ((deprecated("SparseVectorMap::push_back() is deprecated and will be removed in sst 12. please simply use sparsevectormap::insert.")))
```

Comment:
"sparsevectormap::insert" should be "SparseVectorMap::insert()"

Reply:
Fixed